src/plugins/intel_cpu/src/emitters/jit_snippets_emitters.cpp

```diff
@@ -905,8 +935,15 @@ void BrgemmEmitter::emit_brgemm_kernel_call(const brgemm_kernel_t *brgKernel, in
     // todo: Windows ABI : requires different num of arguments passed in regs and on the stack. Need to align.
     h->mov(abi_param1, reinterpret_cast<uintptr_t>(brgKernel));
     h->mov(abi_param2, bs);
-    h->uni_vmovq(abi_param3, Xmm(0));
-    h->uni_vmovq(abi_param4, Xmm(1));
+
+    const auto data_ptr = [&](Xmm xmm, Xbyak::Reg64 reg, size_t memory_bytes_offset, size_t kernel_bytes_offset) {
+        h->uni_vmovq(reg, xmm);
+        if (memory_bytes_offset) h->add(reg, memory_bytes_offset);
+        if (kernel_bytes_offset) h->add(reg, kernel_bytes_offset);
```

Comment:
It's not clear why do we have to distinguish between memory and kernel offset. I think we can just add `load_offset_a` to `in0_offset` before calling `emit_brgemm_kernel_call`. Wouldn't it simplify the logic?

Reply:
Because it's different offsets and I think that a few `add` instructions don't spend a lot of time... But I got your point, so let's unite them. Thanks!